src/Database/Type/DecimalType.php

```diff
@@ -30,23 +30,6 @@ class DecimalType extends Type implements TypeInterface
 {
 
     /**
-     * Identifier name for this type
-     *
-     * @var string|null
-     */
-    protected $_name = null;
-
-    /**
-     * Constructor
-     *
-     * @param string|null $name The name identifying this type
-     */
-    public function __construct($name = null)
-    {
-        $this->_name = $name;
-    }
-
-    /**
```

Comment:
The reason to have this code was so that we could easily remove the inheritance on Type in the near future. I'd rather have it back